docs/source/guides/migration.rst

```diff
@@ -328,6 +328,25 @@ You can run the docker image to start the service.
     2022-07-01T21:57:48+0000 [INFO] [api_server:3] Service loaded from Bento directory: bentoml.Service(tag="iris_classifier:6otbsmxzq6lwbgxi", path="/home/bentoml/bento/")
     2022-07-01T21:57:48+0000 [INFO] [api_server:4] Service loaded from Bento directory: bentoml.Service(tag="iris_classifier:6otbsmxzq6lwbgxi", path="/home/bentoml/bento/")
 
+.. note::
+
+   BentoML 1.x leverage BuildKit <https://github.com/moby/buildkit>`_, to
+   containerize Bentos.
+
+   BuildKit has been `integrated into Docker 18.09 and higher <https://docs.docker.com/develop/develop-images/build_enhancements/>`_.
+   Providing the environment variable ``DOCKER_BUILDKIT=1`` will enable BuildKit:
+
+   .. code-block:: bash
+
+      $ DOCKER_BUILDKIT=1 bentoml containerize iris_classifier:latest
```

Comment:
Can we automatically set this in our CI and not require `buildx`?

Reply:
Our containerize already set this environment variable.